appveyor.yml

```diff
@@ -21,7 +22,7 @@ install:
   # replace with whatever version of volumina triggered the appveyor
   - ps: cp -recurse C:\projects\volumina c:\ilastik\ilastik-meta\volumina
   - timeout 10
-  - python ilastik/scripts/devenv.py create -n %ENV_NAME% -c ilastik-forge conda-forge -p ilastik-dependencies-no-solvers
+  - python ilastik/scripts/devenv.py create -n %ENV_NAME% -c ilastik-forge conda-forge defaults -p ilastik-dependencies-no-solvers
```

Comment:
That is off-topic, but we probably can remove this script since lazyflow is gone, and `conda-develop` is used. It seems that we can replace this with 2 lines of `conda create -n %ENV_NAME% ...` and `conda develop -n %ENV_NAME%`.